src/css/editor.scss

```diff
@@ -1,50 +1,320 @@
-@import 'common';
+@import 'common/common';
+
+
+$editor-width: 450px;
+$editor-font-size: 14px;
+$editor-std-spacing: 15px;
+
+/* -- PP Adder -- */
+.pp-adder {
+    position: absolute;
+    width: 140px;
+    height: 40px;
+    margin-left: -70px;
+    margin-top: -48px;
+
+    .create-annotation {
+        border-color: #2185D0 !important; // override semantic ui
+        color: #2185D0 !important; // override semantic ui
+        cursor: pointer;
+    }
+
+    .create-annotation:hover {
+        color: darken(#2185D0, 10%) !important; // override semantic ui
+    }
+}
+
 
 /* -- PP Editor -- */
+.pp-editor .pp-widget {
+    width: $editor-width;
+    background-color: white;
+    font-size: $editor-font-size;
+    font-family: $light-font-family;
+}
+
+/* --- Editor head bar --- */
+.pp-editor .pp-editor-head-bar {
+    padding: 0 2*$editor-std-spacing 0 $editor-std-spacing;
+    margin: $editor-std-spacing 0 $editor-std-spacing 0;
+
+    .priority-header {
+        display: inline-block;
+        font-family: $bold-font-family;
+        margin: 0 0 10px 0;
+    }
+    /* priority buttons */
+    .pp-editor-priority {
+        display: inline-block;
+        width: calc(100% - 10px);
+        border: none;
+        outline: none;
+        margin: 0;
+        padding: 3px 11px;
+        height: 3em;
+
+        text-align: center;
+        color: rgb(255, 255, 255);
+        font-family: $bold-font-family;
+        text-transform: uppercase;
+
+        background-color: #D8DADA;
+        cursor: pointer;
+    }
+    /* priority-specific styles */
+    .priority-normal, .priority-warning, .priority-alert {
+        display: inline-block;
+        vertical-align: top; //align all buttons at the same level regardless of the number of lines inside
+    }
+    .priority-normal {
+        width: 33%;
+        :hover,
+        .selected {
+            background-color: $normal-ref-color;
+        }
+    }
+    .priority-warning {
+        text-align: center;
+        width: 34%;
+
+        :hover,
+        .selected {
+            background-color: $warning-ref-color;
+        }
+    }
+    .priority-alert {
+        width: 33%;
+        text-align: right;
+
+        :hover,
+        .selected {
+            background-color: $alert-ref-color;
+        }
+    }
+}
+
+/* all form styles */
+.pp-editor {
+    /* fields: comment, referenceLink */
+    .editor-input {
+        display: block;
+        padding: 0;
+        position: relative;
+        margin: 0 2*$editor-std-spacing $editor-std-spacing $editor-std-spacing;
+
+        input, textarea {
+            padding: 0.5em;
+            width: 100%;
+            border: 1px solid #D8DADA;
```

Comment:
Maybe let's move this color to the variables file? I think I've seen it a few times above